src/libraries/Microsoft.Extensions.Caching.Abstractions/src/Microsoft.Extensions.Caching.Abstractions.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
     <EnableDefaultItems>true</EnableDefaultItems>
+    <IsPackable>true</IsPackable>
```

Comment:
Can we instead default this in a common props/targets file?

```xml
<IsPackable Condition="ProjectName.StartsWith('Microsoft.Extensions.') AND $(IsSourceProject)">true</IsPackable>
```

Reply:
IMO I would prefer this property to be explicit